CurrencyExchange_ver2/Bank.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CurrencyExchange_ver2
+{
+    class Bank
+    {
+        double rateUSD, rateEUR, rateRUB;
+        double operationalSum;
+        string currencyType;
+        string exchangeCurrencyType;
+        double resultSumm;
+
+        public Bank() {}
+        public Bank(string request, CurrencyRates rates)
+        {
+            operationalSum = double.Parse(request.Remove(request.Length - 3));
+            currencyType = request.Substring(request.Length - 3, 3);
+            rateUSD = rates.getUSD();
+            rateEUR = rates.getEUR();
+            rateRUB = rates.getRUB();
+        }
+
+        public void SetCurrencyType(string currency)
+        {
+            exchangeCurrencyType = currency;
+        }
+        
+        private void ShaherMaher(Double sum, string sourceCurrency, string finalCurrency)
+        {
+            Double rateOfSource, rateOfFinal;
+            sourceCurrency = sourceCurrency.ToUpper();
+            finalCurrency = finalCurrency.ToUpper();
+
+            if (sourceCurrency.Equals("USD")) rateOfSource = rateUSD;
+            else if (sourceCurrency.Equals("EUR")) rateOfSource = rateEUR;
+            else rateOfSource = rateRUB;
```

Comment:
https://programmerbyday.wordpress.com/2020/11/23/why-you-should-always-use-curly-braces-for-single-statement-blocks-in-csharp/